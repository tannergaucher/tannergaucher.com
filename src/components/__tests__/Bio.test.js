import React from 'react'
import renderer from 'react-test-renderer'
import { PureBio as Bio } from '../Bio'

describe('<Bio/>', () => {
  it(`renders correctly and matches the snapshot`, () => {
    const data = {
      site: {
        siteMetadata: {
          title: 'Tanner Gaucher',
        },
      },
    }
    const tree = renderer.create(<Bio data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
