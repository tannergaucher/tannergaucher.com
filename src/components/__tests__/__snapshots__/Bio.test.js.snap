// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bio/> renders correctly and matches the snapshot 1`] = `
Array [
  <div
    className="sc-jqCOkK etIIiC sc-gqjmRU ebWtEN"
  >
    <span
      aria-label="technologist emoji"
      role="img"
    >
      👨‍💻
    </span>
    Tanner Gaucher
  </div>,
  <div
    className="sc-jqCOkK fXSSaI sc-gqjmRU igXrRB"
  >
    <span
      aria-label="toolbox emoji"
      role="img"
    >
      🧰
    </span>
  </div>,
  <div
    className="sc-jqCOkK fXSSaI sc-gqjmRU igXrRB"
  >
    <span
      aria-label="writing hand emoji"
      role="img"
    >
      ✍️
    </span>
  </div>,
]
`;
